src/command-line/start.js

```diff
@@ -13,6 +13,7 @@ program
 	.option("-B, --bind <ip>", "bind")
 	.option("    --public", "mode")
 	.option("    --private", "mode")
+  .parse(process.argv)
```

Comment:
Actually, nevermind, I found the missing option and added it to #930.

@msaun008, sorry this is a nitpick, but can you replace the 2 spaces with 1 tab please? (Then use `git add src/command-line/start.js && git commit --amend && git push --force` to not create an extra commit)